src/main/java/wolox/training/constants/ExceptionsConstants.java

```diff
@@ -0,0 +1,11 @@
+package wolox.training.constants;
+
+public class ExceptionsConstants {
```

Comment:
A good practice would be to add a private constructor to avoid instantiation of this class since you'll be using it statically :)